client_plugin/drivers/commonUtilDriver/commonUtilDriver.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commonUtilDriver
+
+//
+// Common Utility for driver interface.
+//
+
+import (
+	"github.com/vmware/docker-volume-vsphere/client_plugin/utils/refcount"
+	"path/filepath"
+)
+
+// VolumeDriver - vsphere shared plugin volume driver struct
+type CommonUtilDriver struct {
```

Comment:
the name looks redundant. `commonutilsdriver.CommonUtilsDriver ` ? And why its "commonUtils"  not simply driver or pluginDriver ?